content/docs/introduction/faq.md

```diff
@@ -89,6 +89,16 @@ Yes, with the experimental [Alertmanager](https://github.com/prometheus/alertman
 Yes, with [PromDash](/docs/visualization/promdash/) and [Console
 templates](/docs/visualization/consoles/).
 
+### Can I change the timezone? Why is everything in UTC?
+
+To avoid any kind of timezone confusion, especially when the so-called
+daylight saving time is involved, we decided to exclusively use Unix
```

Comment:
I'll cannot call something by the wrongest name ever. It is neither saving daylight nor a time.

You can change that on your own, if you want. My name will not be spoiled with this excrement of Satan.